hq/test/aws/ec2/EC2Test.scala

```diff
@@ -140,6 +141,22 @@ class EC2Test extends FreeSpec with Matchers with Checkers with PropertyChecks w
         sortedResult should be(sortedResult.sortWith { case ((_, s1), (_, s2)) => s1.size > s2.size })
       }
     }
+
+    "add cloudformation stackId & stackName" in {
```

Comment:
Good to check the happy path, but with only one test we're missing quite a few states.